wasm/jsapi/instance/caching.any.js

```diff
@@ -0,0 +1,67 @@
+// META: global=jsshell
+// META: script=/wasm/jsapi/wasm-module-builder.js
+
+promise_test(async t => {
+  const builder = new WasmModuleBuilder();
+  builder.addImportedMemory("module", "memory", 0, 128);
+  builder.exportMemoryAs("exported");
+  const buffer = builder.toBuffer()
+
+  const memory = new WebAssembly.Memory({ initial: 64, maximum: 128 });
+  const {instance} = await WebAssembly.instantiate(buffer, {
+    module: { memory }
+  });
+  assert_equals(instance.exports.exported, memory);
+}, "Memory");
+
+promise_test(async t => {
+  const builder = new WasmModuleBuilder();
+  builder.addImportedTable("module", "table", 0, 128);
+  builder.addExportOfKind("exported", kExternalTable, 0);
+  const buffer = builder.toBuffer()
+
+  const table = new WebAssembly.Table({ element: "funcref", initial: 64, maximum: 128 });
```

Comment:
```suggestion
  const table = new WebAssembly.Table({ element: "funcref", initial: 0, maximum: 128 });
```
to make the definition of the imported table.